bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -49,13 +49,18 @@
            }
```

Comment:
Yes and no. I wanted to have the event itself be present from the script. This can be useful even without this PR.